scipy/integrate/tests/test_quadpack.py

```diff
@@ -6,6 +6,7 @@
 from numpy import sqrt, cos, sin, arctan, exp, log, pi, Inf
 from numpy.testing import (assert_,
         assert_allclose, assert_array_less, assert_almost_equal)
+import pytest
 from pytest import raises as assert_raises
```

Comment:
Is it reasonable to replace `assert_raises` with `pytest.raises` now that we are importing pytest?

Reply:
Seems reasonable to me

Reply:
Done